src/stylesheets/components/_accordions.scss

```diff
@@ -62,6 +61,7 @@ $accordion-font-family: 'sans' !default;
 .usa-accordion-bordered {
   @include accordion-list-styles;
   @include accordion-nested-list;
+  @include typeset($theme-font-family-accordion);
```

Comment:
Is this bc someone would want the font of the bordered accordion to be different from the default borderless accordion?

Reply:
No, this looks like a mistake

Reply:
nm, this rule is for `.usa-accordion, .usa-accordion-bordered` so it applies to both. I think I can remove the `typeset` from line 18, though

Reply:
Kept this one and removed the one on line 18

Reply:
Oh I see, I think that we should split this up though, to be in line with how we're styling our components, i.e. bordered would need `usa-accordion usa-accordion-bordered` so only the bordered style are added.

Reply:
But should we do it now or in a different PR?

Reply:
Maybe a separate PR — there's a similar issue with `usa-form` and `usa-form-large`

Reply:
Added an issue #2736 to track this work. Add in more as you find them.

Reply:
Ideally, if we could tackle this in the 2.0 PRs we're working on and look for this, that would be good.